3-platform/warehouse/pom.xml

```diff
@@ -27,4 +27,24 @@
     <maven.compiler.target>17</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
   </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.6core.lib</groupId>
+      <artifactId>unit-test-bundle</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-maven-plugin</artifactId>
+          <version>3.1.5</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
```

Comment:
pls dont add code that is not related to the task directly. 
```suggestion
```

Reply:
if you need this for some reason, pls move to the BOM

Reply:
I worked on the issue more deeply. The plugin did not have enough version to work properly. (at order/boot/pom.xml)